mu4/domain/notation/internal/notationinteraction.h

```diff
@@ -96,23 +103,37 @@ class NotationInteraction : public INotationInteraction
 
     Ms::Page* point2page(const QPointF& p) const;
     QList<Element*> hitElements(const QPointF& p_in, float w) const;
+    QList<Element*> elementsAt(const QPointF& p) const;
+    Element* elementAt(const QPointF& p) const;
```

Comment:
So, what is the difference between **hitElements** and **elementsAt** methods?

Reply:
I can’t say, `hitElements` is trying to guess which element we hit, and `elementsAt` accurately determines. I tried not to change the logic.